doc/source/news.rst

```diff
@@ -7,6 +7,79 @@
 News
 ====
 
+.. _release-8-0-2:
+
+Release 8.0.2 - 2018-04-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/executables/grndb`][:ref:`grndb-force-truncate`] Improved
+  ``grndb recover --force-truncate`` option that it can be truncated even if
+  locks are left on the table.
+
+* [Windows] Use ``CONDITION_VARIABLE`` instead of Groonga's original implementation.
+
```

Comment:
It's too close to internal implementation, what is the reason for implementation replacement and what's the benefit from? It seems that "why replaced" and "it solves ..." is important.

Reply:
I changed to a description as below.
```
Fixed a bug that it may also take so much time to change "thread_limit".
```